src/components/SVGViewer/stories/locate.md

```diff
@@ -0,0 +1,31 @@
+# Locate asset in the document
+
+<!-- STORY -->
+
+#### Description:
+
+`isCurrentAsset` callback is responsible for locating the asset and zooming into its area
+
+#### Usage:
+
+```typescript jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { SVGViewer } from '@cognite/gearbox';
+
+function ExampleComponent(props) {
+
+  return (
+    <div style={{ height: '100vh', position: 'relative' }}>
```

Comment:
Is it possible to make a wrapper with `position: relative` inside the component so that users should not have to do that by themselves?

Reply:
fixed